### PR TITLE
SONARHTML-128 Update dependency to SQ 7.9

### PR DESCRIPTION
**Pull request by m-g-sonar**

### CODE REVIEW

File: sonar-html-plugin/src/test/java/org/sonar/plugins/html/lex/PageLexerTest.java [hunk 1]
```diff
@@ -48,13 +46,13 @@ public void testLexer() throws FileNotFoundException {
     PageLexer lexer = new PageLexer();
     List<Node> nodeList = lexer.parse(new FileReader(fileName));
 
-    assertTrue(nodeList.size() > 50);
+    assertThat(nodeList.size() > 50).isTrue();
```
Comment (quentin-jaquier-sonarsource) [inline sonar-html-plugin/src/test/java/org/sonar/plugins/html/lex/PageLexerTest.java:49]:
```suggestion
    assertThat(nodeList.size()).isGreaterThan(50);
```

File: sonar-html-plugin/src/test/java/org/sonar/plugins/html/lex/PageLexerTest.java [hunk 2]
```diff
@@ -132,11 +130,11 @@ public void testDirectiveNode() {
     List<Node> nodeList = new ArrayList<>();
     CodeReader codeReader = new CodeReader(directive);
     tokenizer.consume(codeReader, nodeList);
-    assertEquals(nodeList.size(), 1);
+    assertThat(nodeList).hasSize(1);
     Node node = nodeList.get(0);
-    assertEquals(node.getClass(), DirectiveNode.class);
+    assertThat(DirectiveNode.class).isEqualTo(node.getClass());
```
Comment (quentin-jaquier-sonarsource) [inline sonar-html-plugin/src/test/java/org/sonar/plugins/html/lex/PageLexerTest.java:135]:
```suggestion
    assertThat(node.getClass()).isEqualTo(DirectiveNode.class);
```

File: sonar-html-plugin/src/test/java/org/sonar/plugins/html/rules/HtmlRulesDefinitionTest.java [hunk 3]
```diff
@@ -46,8 +46,9 @@ public void test() {
       .filter(RulesDefinition.Rule::template)
       .map(RulesDefinition.Rule::key)
       .collect(Collectors.toSet());
-    assertThat(templateRules).hasSize(6);
-    assertThat(templateRules).containsOnly("IllegalAttributeCheck",
+    assertThat(templateRules)
+      .hasSize(6)
+      .containsOnly("IllegalAttributeCheck",
```
Comment (quentin-jaquier-sonarsource) [inline sonar-html-plugin/src/test/java/org/sonar/plugins/html/rules/HtmlRulesDefinitionTest.java:51]:
```suggestion
    assertThat(templateRules)
      .containsExactlyInAnyOrder("IllegalAttributeCheck",
```

**Review by quentin-jaquier-sonarsource — APPROVED** [review]
A few suggestions, otherwise, LGTM

File: sonar-html-plugin/src/test/java/org/sonar/plugins/html/lex/PageLexerTest.java (hunk 2 above)
Comment (m-g-sonar) [reply sonar-html-plugin/src/test/java/org/sonar/plugins/html/lex/PageLexerTest.java:135]: even more: `assertThat(node).isInstanceOf(DirectiveNode.class);`

**Pull request merged**